auto_guild.py

```diff
@@ -195,10 +206,12 @@ def transfer_ownership(session, user_id, guild_id):
 
         with open("guild_layout.yaml", "w") as file:
             dump(finished_guild, file)
+        print(f"Guild layout saved to guild_layout.yaml")
```

Comment:
```suggestion
        from pathlib import Path
        pth = Path("./out/")
        pth.mkdir(parents=True, exist_ok=True)
        with open(str(pth / "guild_layout.yaml"), "w") as file:
            dump(finished_guild, file)
        print(f"Guild layout saved to ./out/guild_layout.yaml")
```
Should output files be put in a dedicated directory, so that they don't get thrown in the middle of the rest of the files?